src/validation/services/subject.js

```
@@ -58,7 +59,7 @@ const validateCategoryOnUpdate = async (subjectId, updateData) => {
 }
 
 const validateSubjectOnCreate = async (subjectData) => {
-  const { name, category } = subjectData;
+  const { name, category, description } = subjectData;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)